chpl/chpl-service/pom.xml

```diff
@@ -424,14 +424,6 @@
                             <source>src/main/java/gov/healthit/chpl/domain/TransparencyAttestationMap.java</source>
                         </sources>
                     </configuration>
-                    <executions>
```

Comment:
Don't we need schemagen to generate the download file XSD? Why remove it?

Reply:
I've lost track. Are we still using schemagen? Or is that no longer needed due to the modifications we made to the XML generation system?

Reply:
We still use it to generate the XSD. I confirmed by checking Bamboo and see this line in one of the steps:
`cp chpl/chpl-service/target/generated-resources/schemagen/schema1.xsd /opt/tomcat8/tomcat-latest/files
`